src/system/RoutesModule/Form/Type/Base/AbstractRouteType.php

```diff
@@ -276,27 +276,25 @@ public function addReturnControlField(FormBuilderInterface $builder, array $opti
     /**
      * Adds submit buttons.
      *
-     * @param FormBuilderInterface $builder The form builder.
-     * @param array                $options The options.
+     * @param FormBuilderInterface $builder The form builder
+     * @param array                $options The options
      */
     public function addSubmitButtons(FormBuilderInterface $builder, array $options)
     {
         foreach ($options['actions'] as $action) {
             $builder->add($action['id'], 'Symfony\Component\Form\Extension\Core\Type\SubmitType', [
-                'label' => $this->__(/** @ignore */$action['title']),
+                'label' => $this->__($action['title']),
```

Comment:
ftr, the `@ignore` tag doesn't prevent translation, it only prevents the translator extractor from  extracting the string that follows it (e.g. `$action['title']` literal)